.github/workflows/build_dev.yaml

```diff
@@ -8,7 +8,10 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v1
-      - name: Bygg app med yarn
-        run: |
-          yarn
-          yarn build
\ No newline at end of file
+      - uses: actions/checkout@v1
```

Comment:
Ser ut som du kjører checkout to ganger her?

Reply:
Takk! 
```suggestion
```